muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/WorkspaceMaterialFieldAnswerController.java

```diff
@@ -231,25 +241,13 @@ public WorkspaceMaterialFileFieldAnswerFile findWorkspaceMaterialFileFieldAnswer
   }
   
   public void deleteWorkspaceMaterialFileFieldAnswerFile(WorkspaceMaterialFileFieldAnswerFile fieldAnswerFile) {
+    if (fileAnswerUtils.isFileSystemStorageEnabled()) {
+      Long userEntityId = fieldAnswerFile.getFieldAnswer().getReply().getUserEntityId();
+      fileAnswerUtils.removeFileFromFileSystem(userEntityId, fieldAnswerFile.getFileId());
```

Comment:
Should this check if the file is even expected to be there? (it does silently skip it though if not)

Reply:
Adding existence check and exception if file exists but removal fails.